circle.yml

```diff
@@ -75,12 +76,16 @@ deployment:
       - |
         test -z "${DOCKER_USER}" || (
           docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS &&
-          (test "${DOCKER_ORGANIZATION:-$DOCKER_USER}" == "weaveworks" || (
-            docker tag weaveworks/scope:latest ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:latest &&
-            docker tag weaveworks/scope:$(./tools/image-tag) ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:$(./tools/image-tag)
+          (test "${DOCKER_ORGANIZATION:-$DOCKER_USER}" != "weaveworks" && (
```

Comment:
Can you explain briefly why `== "weaveworks" ||` was changed to `!= "weaveworks" &&`?